M2/Macaulay2/m2/help.m2

```diff
@@ -249,7 +249,9 @@ documentationValue(Symbol, Package)         := (S, pkg) -> if pkg =!= Core then
 	    cert  = new HashTable from cert;
 	    -- TODO: compare with the one in installPackage.m2
 	    star := IMG { "src" => replace("PKG", "Style",currentLayout#"package") | "GoldStar.png", "alt" => "a gold star"};
-	    commit := replace("(?<=/blob/)master", toString cert#"release at publication", cert#"repository code URI");
+	    commit := concatenate(
+		"https://github.com/Macaulay2/M2/blob/",
+		cert#"release at publication", "/M2/Macaulay2/packages/", fn);
```

Comment:
But someone might download this file thinking they can use it, and find it unusable, no?

Reply:
Ah you're right. I still think this should be changed, but that's separate from this PR.